roles/openshift_node/tasks/distribute_bootstrap.yml

```diff
@@ -41,25 +41,3 @@
       msg: "{{ logs_node.stdout_lines }}"
   - fail:
       msg: Node start failed.
-
-# The restart above triggers previously approved nodes to go NotReady
```

Comment:
This is no longer necessary; previously provisioned nodes (and masters) will no longer be restarted because they won't be in scope.